src/odf/metadata/src/serde/flatbuffers/proxies_generated.rs

```diff
@@ -1608,10 +1618,18 @@ impl<'b> flatbuffers::Push for Timestamp {
     type Output = Timestamp;
     #[inline]
     unsafe fn push(&self, dst: &mut [u8], _written_len: usize) {
-        let src =
-            ::core::slice::from_raw_parts(self as *const Timestamp as *const u8, Self::size());
+        let src = ::core::slice::from_raw_parts(
+            self as *const Timestamp as *const u8,
+            <Self as flatbuffers::Push>::size(),
+        );
         dst.copy_from_slice(src);
     }
+
+    // FIXME: https://github.com/kamu-data/kamu-cli/issues/1084
+    // #[inline]
+    // fn alignment() -> flatbuffers::PushAlignment {
+    //     flatbuffers::PushAlignment::new(4)
+    // }
```

Comment:
Note: Manual change to preserve compatibility with old blocks, see [ticket](https://github.com/kamu-data/kamu-cli/issues/1084) for details